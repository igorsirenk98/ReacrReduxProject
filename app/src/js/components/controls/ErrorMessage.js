import React from 'react';

import CssBaseline from '@material-ui/core/CssBaseline';
import Typography from '@material-ui/core/Typography';
import Container from '@material-ui/core/Container';

const ErrorMessage = (props) => {
    const { message } = props.error;

    return (
        <>
            <CssBaseline />
            <Container className="container" fixed>
                <Typography className="notification" gutterBottom variant="h5" component="h2">
                    Error! {message}
                </Typography>
            </Container>
        </>
    );
}

export default ErrorMessage;
