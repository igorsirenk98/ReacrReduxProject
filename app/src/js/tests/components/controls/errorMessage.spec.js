import React from 'react'
import { shallow } from 'enzyme'
import ErrorMessage from '../../../components/controls/ErrorMessage';

describe('ErrorMessage component', () => {
    const errorMessage = shallow(<ErrorMessage error={{ message: 'error' }} />);

    it('renders correctly', () => {
        expect(errorMessage.debug()).toMatchSnapshot()
    })
})
