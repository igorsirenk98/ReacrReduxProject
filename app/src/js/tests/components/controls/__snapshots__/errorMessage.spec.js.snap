// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage component renders correctly 1`] = `
"<Fragment>
  <CssBaseline />
  <WithStyles(ForwardRef(Container)) className=\\"container\\" fixed={true}>
    <WithStyles(ForwardRef(Typography)) className=\\"notification\\" gutterBottom={true} variant=\\"h5\\" component=\\"h2\\">
      Error! 
      error
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(Container))>
</Fragment>"
`;
